### ISSUE TITLE
Accuracy issue

### ISSUE THREAD

**Issue by NomiMalik0207** [moskomule/mixup.pytorch#3]
Hi i am using this data augumentation technique on my own dataset. It solve the problem of overfitting but accuracy remains same?  How can it improve the classification accuracy?

**Comment by moskomule** [comment]
Hi, I think it depends on many hyperparameters e.g. data size, network.

This is a tip by the author of mixup: https://github.com/moskomule/mixup.pytorch/issues/1 I hope it will help you.

**Comment by NomiMalik0207** [comment]
i am using VGG16. now i am trying 0.2 aur 0.4 alpha size. 
on the other han 35000 thousand images are in training set and 3500 are randomly selected for testing.

**Comment by moskomule** [comment]
How about ResNet or others?

**Comment by NomiMalik0207** [comment]
I test it on ResNet and test accuracy is improved about 1%. but its due to resnet is more deep?
